### PR TITLE
Release redisenterprise 1.2.1b1

### PR DESCRIPTION
**Pull request by priyankarking**
making latest version as preview

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by wangzelin007 — APPROVED** [review]

File: src/redisenterprise/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+1.3.0b1
++++++++++
+- Fix: Make latest version as peview as last 2 versions(1.2.0 abd 1.1.0) are mistakenly not marked as preview.
```
[the remark this answers is not in the document]
Comment (AllyW) [reply src/redisenterprise/HISTORY.rst:7]: I prefer to release a new preview version 1. 2.1b1.

[the remark this answers is not in the document]
Comment (AllyW) [reply src/redisenterprise/HISTORY.rst:7]: And @priyankarking  if preview version is expected, `azext.isPreview: true` in azext_metadata.json is required for all preview extension module releases. 

For more info, please refer to doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#backward-compatibility-of-extension-version-tags

[the remark this answers is not in the document]
Comment (priyankarking) [reply src/redisenterprise/HISTORY.rst:7]: Our service is GA and extension is not preview mode. Its just that latest version is based on preview API. Older versions might not be preview.

So, I don't it is required to change the azext_metadata.json

[the remark this answers is not in the document]
Comment (priyankarking) [reply src/redisenterprise/HISTORY.rst:7]: @AllyW  ^^

[the remark this answers is not in the document]
Comment (AllyW) [reply src/redisenterprise/HISTORY.rst:7]: hi @priyankarking cli extension module's versioning is not bonded to the service api version, and its versioning schema is defined here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#versioning-of-azure-cli-extension

If we release a new `preview` version, like `1.3.0b1` here, then, `azext.isPreview` tag should be set as `true`, to keep backward compatibility (https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#backward-compatibility-of-extension-version-tags).  And `1.3.0b1` means `preview` version of `1.3.0`, `b` means `pre` or `preview` of stable version.

If the new release is `stable`, for example `1.3.0`, then, `azext.isPreview:true` is unnecessary.

**Review by wangzelin007 — APPROVED** [review]

**Review by AllyW — APPROVED** [review]

**Pull request merged**